libbeat/_meta/fields.ecs.yml

```diff
@@ -97,6 +118,125 @@
             different values which are then freely searchable. If multiple
             messages exist, they can be combined into one message.
     
+    - name: client
```

Comment:
I'm not fully convinced we should add the full ecs fields yml to each Beat or only add the fields which are used by more then one Beat. We can still clean this up later.